### PR TITLE
Docs: Add example fetching keyword in top_metrics

### PR DESCRIPTION
**Pull request by nik9000**
Adds an example of fetching a keyword field.

### CODE REVIEW

File: docs/reference/aggregations/metrics/top-metrics-aggregation.asciidoc [hunk 1]
```diff
@@ -71,6 +72,7 @@ request. So,
 `metrics` selects the fields of the "top" document to return. You can request
 a single metric with something like `"metric": {"field": "m"}` or multiple
 metrics by requesting a list of metrics like `"metric": [{"field": "m"}, {"field": "i"}`.
+The fields can be <<numbers,number>>, <<keywords,keyword>>, or <<ips,ip>>.
```
Comment (jrodewig) [inline docs/reference/aggregations/metrics/top-metrics-aggregation.asciidoc:75]: These xrefs are breaking the docs CI. This should fix them up.

```suggestion
The fields can be <<number,numeric>>, <<keyword,keyword>>, or <<ip,`ip`>>.
```

**Review by jrodewig — APPROVED** [review]
LGTM. Thanks for this fix @nik9000.

**Review by dgieselaar — APPROVED** [review]
Thanks Nik! I missed the fact that that line only applied to sorting.

File: docs/reference/aggregations/metrics/top-metrics-aggregation.asciidoc (hunk 1 above)
Comment (nik9000) [reply docs/reference/aggregations/metrics/top-metrics-aggregation.asciidoc:75]: :+1:

**Pull request merged**